diff --git a/src/Http.php b/src/Http.php
index 715bc11..4c2b01f 100644
--- a/src/Http.php
+++ b/src/Http.php
@@ -415,10 +415,10 @@
         // Parse $_POST.
-        if ($_SERVER['REQUEST_METHOD'] === 'POST' && $_SERVER['CONTENT_TYPE']) {
+        if ($_SERVER['REQUEST_METHOD'] === 'POST') {
             match ($_SERVER['CONTENT_TYPE']) {
                 'multipart/form-data' => static::parseMultipart($http_body, $http_post_boundary),
                 'application/json' => $_POST = \json_decode($http_body, true, flags: \JSON_THROW_ON_ERROR) ?? [],
                 'application/x-www-form-urlencoded' => \parse_str($http_body, $_POST),
-                default => ''
+                default => $_POST = \json_decode($http_body, true) ?? []
             };
         }
 }
